### PR TITLE
to_cs_primitive: add missing types

### PR DESCRIPTION
**Pull request by yjh0502**

### CODE REVIEW

**Comment by toolness** [comment]
Wooot thank you!

File: src/lib.rs [hunk 1]
```diff
@@ -534,11 +534,18 @@ fn munge_cs_name(name: String) -> String {
 
 fn to_cs_primitive<'a>(type_name: &'a str) -> &'a str {
     match type_name {
-        "u8" => "byte",
-        "f32" => "float",
+        "i8" => "Byte",
+        "u8" => "SByte",
```
Comment (toolness) [inline src/lib.rs:538]: Wait, shouldn't these be the other way around?  Isn't an `SByte` a signed byte, while a `Byte` is an unsigned one?

According to [the Microsoft docs](https://docs.microsoft.com/en-us/dotnet/api/system.sbyte?view=netframework-4.8):

> The SByte value type represents integers with values ranging from negative 128 to positive 127.

Comment (toolness) [reply src/lib.rs:538]: I fixed this in dfe874de77a180d28210faac6513fbb5b0eb05aa but let me know if you disagree!

Comment (yjh0502) [reply src/lib.rs:538]: That's my fault :( Thanks for catching the error!

Comment (yjh0502) [reply src/lib.rs]: That's my fault :( Thanks for catching the error!

Comment (toolness) [reply src/lib.rs:538]: No worries! :)

**Pull request merged**